libjob_queue/tests/job_lsf_exclude_hosts_test.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2016  Statoil ASA, Norway.
+ *
+ * This file is part of ERT - Ensemble based Reservoir Tool.
+ *
+ * ERT is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+ * for more details.
+ */
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <unistd.h>
+
+#include <assert.h>
+#include <ert/util/util.h>
+
+#include <ert/job_queue/lsf_driver.h>
+#include <ert/job_queue/lsf_job_stat.h>
+
+void test_submit(lsf_driver_type * driver, const char * cmd) {
+  assert(lsf_driver_set_option(driver , LSF_DEBUG_OUTPUT , "TRUE" ));
+  assert(LSF_SUBMIT_INTERNAL == lsf_driver_get_submit_method(driver));
```

Comment:
Hmmm - I can not understand that this will trigger? And - anyway it should not be necessary?!